plugins/rtmp-services/data/services.json

```diff
@@ -2484,6 +2484,26 @@
                 "keyint": 2,
                 "max video bitrate": 16000
             }
+        },
+        {
+            "name": "Vindral",
+            "more_info_link": "https://docs.vindral.com/docs/vindral-cdn/",
+            "servers": [
+                {
+                    "name": "eu-north",
+                    "url": "rtmps://rtmp.eu-north.cdn.vindral.com/publish"
+                },
+                {
+                    "name": "eu-west",
+                    "url": "rtmps://rtmp.eu-west.cdn.vindral.com/publish"
+                }
+            ],
+            "recommended": {
+                "keyint": 1,
+                "profile": "baseline",
```

Comment:
Are you sure about this? The quality hit from limiting to baseline is fairly considerable.

Additionally, you may want to consider setting a maximum audio and video bitrate.

Reply:
Many of our customers still target devices that only support baseline. As long as the option to select a different profile exists we believe this is the best setting for us.

We don't have limits for max video and audio bitrate but added high numbers for sanity check

Reply:
I understand, in our case it's not really misguided. We have cases where `baseline` is preferred. But after some more consideration we've decided to go with `high` instead, for the same reasons that you've already stated